pkg/client/options_test.go

```diff
@@ -330,6 +330,12 @@ var _ = Describe("ForceOwnership", func() {
 		t.ApplyToSubResourcePatch(o)
 		Expect(*o.Force).To(BeTrue())
 	})
```

Comment:
nit: I think there are some tests missing for the other options that we support

Reply:
Done